sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ArithmeticExpressionSuite.scala

```diff
@@ -1086,4 +1086,17 @@ class ArithmeticExpressionSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(IntegralDivide(Literal(Duration.ofDays(1)),
       Literal(Duration.ofHours(-5))), -4L)
   }
+
+  test("unit test for gatherCommutative()") {
```

Comment:
can we move this test to `CanonicalizeSuite`?

Reply:
Done